### PR TITLE
editor: add open res folder to project menu

### PR DESCRIPTION
**Pull request by Fishrock123**
This is a bit more consistent with most other tools (e.g. "File" menus).

Lmk if there's anything else I need to do here! :D

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -5456,6 +5460,7 @@ EditorNode::EditorNode() {
 	p->add_separator();
 
 	p->add_item(TTR("Open Project Data Folder"), RUN_PROJECT_DATA_FOLDER);
+	p->add_item(TTR("Open Project Resource Folder"), RUN_PROJECT_FOLDER);
```
Comment (Fishrock123) [inline editor/editor_node.cpp]: I originally called this "Open Project Folder" but I guess this naming is more descriptive/accurate.

**Comment by akien-mga** [comment]
That's the same as doing Right Click > Show in File Manager in the FileSystem dock. I don't know if it's worth adding another entry to the Project menu for that, but I'm not strongly opposed to it either. Opinions?

**Comment by reduz** [comment]
I moved this to Tools in my PR #27781, so please wait until it gets merged, should be moved there too.

**Comment by akien-mga** [comment]
> I moved this to Tools in my PR #27781, so please wait until it gets merged, should be moved there too.

This could now be rebase, that PR was merged.

**Comment by akien-mga** [comment]
Bump.

**Comment by akien-mga** [comment]
Closing for now as the PR can't be merged as is. Feel free to open a new PR with a rebased version if it is still relevant.

**Pull request closed without merging**